db/builder.cc

```diff
@@ -281,6 +282,28 @@ Status BuildTable(
       if (s.ok()) {
         s = blob_file_builder->Finish();
       }
+      assert(blob_file_additions);
+      assert(!ioptions.cf_paths.empty());
+#ifndef ROCKSDB_LITE
```

Comment:
Ah ok. Thanks for pointing it out. I will check and update the both locations. I added here because Blob file builder calls Finish(). I didn't consider about failure.